pykeepass/__init__.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import
 from .pykeepass import PyKeePass, create_database
+from .kdbx_parsing.factorinfo import FactorInfo, FactorGroup, FIDO2Factor, PasswordFactor, KeyFileFactor
 
 from .version import __version__
 
-__all__ = ["PyKeePass", "create_database", "__version__"]
+__all__ = ["PyKeePass", "create_database", "__version__", 'FactorInfo', 'FactorGroup', 'FIDO2Factor', 'KeyFileFactor']
```

Comment:
As these classes are necessary to make a database *start* using multifactor auth, I figured it made sense to expose them at the top level.